src/radio-button/index.css

```diff
@@ -0,0 +1,11 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021 Gemeente Utrecht
+ */
+
+.utrecht-radio-button {
+  margin-block-start: 0;
```

Comment:
#moveTheMargins

Reply:
This is to reset a native margin of 3px, I'll add a comment to clarify.